docs/step-by-step/kubeapps-on-tkg/step-3.md

```diff
@@ -1,138 +1,115 @@
+> **NOTE**: This repository is currently under heavy development. Therefore, the URL used below is subject to change.
 
-## 3.2 - Adding the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced
+The public content from the VMware Marketplace™ repository can be retrieved at `https://charts.market.csp.vmware.com/bitnami`. Since this is a public repository, it is only necessary to configure the following values in the _Add An App Repository_ form:
 
-The private content from the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced can be retrieved at `https://registry.pivotal.io/tac-for-tanzu-advanced/charts/.`.
-Since it is a private repository, it requires adding the `Basic Auth` with username and password. If you do not have it, please reach out to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html).
-Furthermore, it is an OCI registry; it implies that you have to manually add the list of artifacts to retrieve.
+- _Name_: Add a descriptive name, such as `vmware-marketplace`
+- _URL_: Use the endpoint URL `https://charts.market.csp.vmware.com/bitnami`
 
-The configuration can be summarized as follows:
+![Adding the public repository](./img/step-3-8.png)
 
-1. **Name**: add descriptive name, for instance, `tac-for-tanzu-advanced`
-2. **URL**: set it to `https://registry.pivotal.io/tac-for-tanzu-advanced/charts/`
-3. **Repository Authorization**: set it to `Basic Auth`
-   - Enter your VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced `username` and `token` in the _Username_ and _Password_ fields respectively.
-4. **Repository Type**: set it to `OCI Registry`
+Once complete, the public catalog will be visible in Kubeapps, as shown below:
 
-   - Add the following data in the _List of Repositories_ field:
+![Catalog after adding the public repository](./img/step-3-9.png)
 
-   ```
-   kibana, kiam, mariadb, postgresql-ha, grafana-operator, nginx-ingress-controller, mongodb, minio, nginx, apache, ejbca, mongodb, elasticsearch, etcd, solr, kubewatch, tomcat, kong, rabbitmq, grafana, logstash, jenkins, postgresql, redis, kube-state-metrics, airflow, mariadb-galera, contour, influxdb, cassandra, wavefront, nats, aspnet-core, wordpress, tensorflow-resnet, spring-cloud-dataflow, fluentd, zookeeper, kafka, node-exporter, memcached, redis, parse, keycloak, external-dns, pytorch, mxnet, harbor, thanos, spark, consul, kubeapps, mysql, wildfly, metrics-server
-   ```
+### Add the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced
 
-> **TIP**: To check the progress of the synchronization task, use the following command:
->
-> ```bash
-> kubectl logs -n kubeapps -l apprepositories.kubeapps.com/repo-name=tac-for-tanzu-advanced
-> ```
+The private content from the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced repository can be retrieved at `https://registry.pivotal.io/tac-for-tanzu-advanced/charts/.`. Since this is a private OCI registry, there are two key differences to note in the process:
```

Comment:
Please recheck dot at end of URL

Reply:
Oops, leftover. It should be `https://registry.pivotal.io/tac-for-tanzu-advanced/charts`
Thanks!